src/Symfony/Component/Workflow/MarkingStore/PropertyAccessorMarkingStore.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Workflow\MarkingStore;
+
+use Symfony\Component\PropertyAccess\PropertyAccess;
+use Symfony\Component\PropertyAccess\PropertyAccessorInterface;
+use Symfony\Component\Workflow\Marking;
+
+/**
+ * PropertyAccessorMarkingStore.
+ *
+ * @author Grégoire Pineau <lyrixx@lyrixx.info>
+ */
+class PropertyAccessorMarkingStore implements MarkingStoreInterface
+{
+    /**
+     * @var string
+     */
+    private $property;
+
+    /**
+     * @var PropertyAccessorInterface
```

Comment:
:+1: I hate almost all docblock. It was not added by me but it's with great pleasure I'm going to remove it ;)